### DISCUSSION TITLE
Persisting configuration

### DISCUSSION THREAD

**Discussion by asturl** [bluenviron/mediamtx#2917] [Questions] [answered]
### Question

It used to be possible to query the current configuration via e.g.
`/v2/config/get`
to get a json response which allowed me to overwrite the server's .yml file without worrying about the full config/API specification on my code side.
This way, any configuration or path changes I made via the API could easily be persisted.

Now in v3, the API was split into global configuration and path configuration, so this is no longer possible.

Am I just blind or is there no inherent/intentional way to persist the configuration?

**Comment by aler9** [comment, marked as answer]
Persisting API changes is currently not implemented, the feature can be tracked here #1406
